imagedephi/redact/redact.py

```diff
@@ -54,8 +54,8 @@ def redact_images(
 ) -> None:
     base_rules = get_base_rules()
     images_to_redact = iter_image_files(input_path) if input_path.is_dir() else [input_path]
-    time_stamp = datetime.datetime.now().isoformat(timespec="seconds")
-    redact_dir = Path(f"{output_dir}/Redacted_{time_stamp}")
+    time_stamp = datetime.datetime.now().isoformat(timespec="seconds").replace(":", "")
```

Comment:
This time format is still a bit hard for me to read. How about:

```suggestion
    time_stamp = datetime.datetime.now().strftime("%y-%m-%d-%H-%M-%S")
```

See:
* https://docs.python.org/3/library/datetime.html#datetime.datetime.strftime
* https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes

You could also decide to make the year be 4 digits. I don't have a strong preference.

This also should be more predictable if the `datetime.now()` has `microsecond` precision.

Reply:
changed to this:
`Redacted_2023-05-15_15-58-58`
to me its easier to distinguish date from time with the underscore